src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Arm/Sve.PlatformNotSupported.cs

```diff
@@ -1423,6 +1423,187 @@ public new abstract class Arm64 : AdvSimd.Arm64
         /// </summary>
         public static unsafe Vector<ulong> LoadVectorUInt32ZeroExtendToUInt64(Vector<ulong> mask, uint* address) { throw new PlatformNotSupportedException(); }
 
+        /// <summary>
+        /// svuint8x2_t svld2[_u8](svbool_t pg, const uint8_t *base)
+        ///   LD2B {Zresult0.B, Zresult1.B}, Pg/Z, [Xbase, #0, MUL VL]
+        /// </summary>
+        public static unsafe (Vector<byte>, Vector<byte>) LoadVectorx2(Vector<byte> mask, byte* address) { throw new PlatformNotSupportedException(); }
```

Comment:
https://github.com/dotnet/runtime/issues/99957 is unfortunately not tracking this quite correctly.

The approved API names are from https://github.com/dotnet/runtime/issues/97831 and are `Load2xVector`, `Load3xVector`, and `Load4xVector`

These were decided upon as API review felt that `Vectorx` was too confusing.

Reply:
API review also asked that `AdvSimd.LoadVector128x2` and friends be renamed to follow the same scheme since they haven't shipped yet, thus should be named `Load2xVector128` instead (for example).

It'd be great if we could get that handled either in this PR or in a simultaneous PR (CC. @kunalspathak since you did the original work around those APIs)

Reply:
What about `Storex3`? Should that be also `Store2xVector` or equivalent? #102262

Reply:
> API review also asked that `AdvSimd.LoadVector128x2` and friends be renamed to follow the same scheme since they haven't shipped yet, thus should be named `Load2xVector128` instead (for example).

Is there any reason of changing these as well? They clearly differentiate the lower `x` from the size. Is the idea just so they are consistent with SVE APIs?

I think that should also apply to AdvSimd Store* too, e.g. `StoreVector128x2AndZip` should be `Store2xVector128AndZip` and `StoreVector128x2` should be `Store2xVector128` and so forth?

Reply:
> Is there any reason of changing these as well? 

Its a consistency thing. It would be a different consideration if they had already shipped, but they haven't.

> I think that should also apply to AdvSimd Store* too, 

Yes. Those should be just `Store` as the type of store is determined by the input parameter type (overload resolution). We only differentiate by name when the behavior significantly differs or the APIs cannot be disambiguated by overload.

Reply:
I have opened https://github.com/dotnet/runtime/issues/102340 to track the renaming of AdvSimd/SVE. However, since this PR is already adding Load APIs, we should fix the new naming in this PR itself. @TIHan - the API names should be `Load2xVectorAndUnzip`, `Load3xVectorAndUnzip` and so forth.

Reply:
Can we do that in a separate PR so this one doesn't touch the other APIs?

Reply:
> @TIHan - the API names should be Load2xVectorAndUnzip, Load3xVectorAndUnzip and so forth.

This is applicable to the SVE APIs that are being added in this PR.